plugins/Feedback/lang/en.json

```diff
@@ -47,7 +47,7 @@
         "ReferBannerTitle": "Don't let your friend's data end up in the wrong hands!",
         "ReferBannerLonger": "Refer them to Matomo Analytics now to take back control!",
         "ReferBannerEmailShareSubject": "Refer them to Matomo Analytics now to take back control!",
-        "ReferBannerEmailShareBody": "I choose Matomo, an ethical alternative to Google Analytics that gives me 100% data ownership and protects the data of my website visitors.\r\nI’m sharing this message in the hope that you too will take back the power from Google and get complete ownership of your own data.\r\n\r\nCheck out Matomo at https://matomo.org",
+        "ReferBannerEmailShareBody": "I choose Matomo, an ethical alternative to Google Analytics that gives me 100%% data ownership and protects the data of my website visitors.\r\nI’m sharing this message in the hope that you too will take back the power from Google and get complete ownership of your own data.\r\n\r\nCheck out Matomo at https://matomo.org",
```

Comment:
A simple `%` in translations needs to be escaped with another `%`. There was actually also one test failing because of this.